developer-docs-site/docs/reference/node_liveness_criteria.md

```diff
@@ -0,0 +1,52 @@
+
+This document describes how you can verify the status of your deployed validator node in the incentivized testnet to meet the success criteria.
+
+# Liveness criteria
+
+The liveness of your validator node will be evaluated using both on-chain and off-chain data. On-chain data will be pulled directly from nodes syncing to the chain, and off-chain data will be shipped via telemetry. Such data includes:
+
+- At least one proposed block per hour. This will be used to determine your node’s availability over time.
+- Node pushing telemetry data, which includes the below metrics
+- A continuously increasing synced version of your node, alongside a reasonable delta from the highest state of the blockchain
+- Aptos Validator is among your set of peers
+
+# Verifying the liveness of your node
+
+## Locally
+
+If you are a node operator, then several tools are available to you (provided by the Aptos team and the community) to verify the status of your own node locally. This local status will act as a good proxy for overall node health as seen from the network level and reported in a remote system operated by Aptos Labs. 
+
+- Locally, the best way to verify your node status is to interact with your node. You can monitor your local metrics endpoint by running a `curl` command and observe various key metrics. Follow the steps described in detail in the [Verify initial synchronization](https://aptos.dev/tutorials/full-node/run-a-fullnode/#verify-the-correctness-of-your-fullnode) document.
+
+:::tip
+
+When you register your validator node for incentivized testnet, you will be prompted to perform this check.
+
+:::
+
+
+- To make your validator node more observable, you may install monitoring tools that scrape the local metrics endpoint:
+    - For Kubernetes based deployments, install the monitoring Helm chart ([https://github.com/aptos-labs/aptos-core/tree/main/terraform/helm/monitoring](https://github.com/aptos-labs/aptos-core/tree/main/terraform/helm/monitoring))
+    - Locally, you may run Prometheus and Grafana directly. Dashboards that utilize the metrics can be found here: ([https://github.com/aptos-labs/aptos-core/tree/main/dashboards](https://github.com/aptos-labs/aptos-core/tree/main/dashboards))
+
+The two above monitoring methods rely on your node’s reported Prometheus Metrics. Of particular importance, the following metrics are directly related to the liveness success criteria above:
+
+- `aptos_consensus_proposals_count`
+- `aptos_state_sync_version{type="synced"}`
+- `aptos_connections`
+
+## Remotely
+
+Remotely, the Aptos team can verify the state of your node via [telemetry](/reference/telemetry). 
```

Comment:
Change the link format to `[telemetry](telemetry.md)` because the telemetry.md file is in the cwd. The other day @zacharydenton figured out that this format is more platform- and browser-agnostic.